.github/workflows/rails.yml

```diff
@@ -0,0 +1,45 @@
+name: Rails
+on: [push, pull_request]
+jobs:
+  Rails-Tests:
+    runs-on: ubuntu-latest
+
+    services:
+      postgres:
+        image: postgres:11
```

Comment:
we're using postgres 9.5 in production. 
maybe use same version as in prod here?

Reply:
Yes totally agree